great_expectations_cloud/agent/agent.py

```diff
@@ -513,6 +519,12 @@ def _create_scheduled_job_and_set_started(
                     "organization_id": str(org_id),
                 },
             )
+        if not response.status_code != 201:  # noqa: PLR2004
```

Comment:
This is overly specific status code handling.
see 👉  https://github.com/great-expectations/cloud/pull/481/files#r1801193270